tests/test_spidermiddleware_output_chain.py

```diff
@@ -292,7 +292,7 @@ def crawl_log(self, spider):
         crawler = get_crawler(spider)
         with LogCapture() as log:
             yield crawler.crawl(mockserver=self.mockserver)
-        raise defer.returnValue(log)
```

Comment:
I suspect the raise expression here was a mistake and it was ignored, so this should be rewritten as `return log`.

Reply:
@wRAR  return log is leading to a lot of errors and test failures in my local machine.

Reply:
@joybhallaa it doesn't on mine.

Reply:
That's cool, just thought I'd let you know.

Reply:
@joybhallaa that probably means you changed something else.